src/main/java/com/flipkart/foxtrot/client/senders/HttpSyncEventSender.java

```diff
@@ -45,25 +45,36 @@ public HttpSyncEventSender(final FoxtrotClientConfig config, FoxtrotCluster clie
     }
 
     @Override
-    public void send(Document document) {
-        send(Collections.singletonList(document));
+    public void send(Document document) throws Exception {
+        send(table, document);
     }
 
     @Override
-    public void send(List<Document> documents) {
+    public void send(String table, Document document) throws Exception {
```

Comment:
That is the whole point of this patch. Otherwise people will not be able to
push to multiple apps. This specially becomes a problem in ingestion
systems.
On 31-May-2016 3:35 PM, "Rishabh" <notifications@github.com> wrote:

> In
> src/main/java/com/flipkart/foxtrot/client/senders/HttpSyncEventSender.java
> <https://github.com/flipkart-incubator/foxtrot-client/pull/8#discussion_r65154551>
> :
>
> >      }
> >
> >      @Override
> > -    public void send(List<Document> documents) {
> > +    public void send(String table, Document document) throws Exception {
>
> This could become confusing for user. There is already a table at Sender
> level.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/flipkart-incubator/foxtrot-client/pull/8/files/9c40282297f37bfb318827310537f9453ccf84dc#r65154551>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABhO591gODK7px2jl67Xyo27UXFFdWTbks5qHAf6gaJpZM4IqRME>
> .
>